extensions/depthmap2mask/install.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
+import os
+
+git = os.environ.get("GIT", "git")
+
+
+def git_clone(url, dir, name, commithash=None):
```

Comment:
Copied over code from the `launch.py` file.  Errors were being thrown when trying to import the module.  Messy stuff so it's best to just do it this way for now.